spec/features/auto_deploy_spec.rb

```diff
@@ -1,5 +1,8 @@
 require 'rails_helper'
 
+require 'sidekiq/testing'
```

Comment:
Can you put both of these lines in `spec/support/sidekiq.rb`? That will be required at boot time when running the tests.